### PR TITLE
Release30

### PR DESCRIPTION
**Pull request by CaroMac**
contains properties get / set / delete cmd and release notes

### CODE REVIEW

**Review by KirbyKatcher — DISMISSED** [review]

**Review by eamansour — CHANGES_REQUESTED** [review]

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+title: "Managing automated test runs"
+---
+
+[Retrieving properties](#retrieving)<br>
+[Setting properties](#setting)<br>
+[Deleting properties](#deleting)<br>
+
+The ability to retrieve, set, and delete properties held in the configuration properties store (CPS) directly into an Ecosystem by using the Galasa CLI makes it easier for testers to set parameters and credentials on the Ecosystem for tests to read and use at runtime. 
+
+System administrators can use the CLI to set Ecosystem-wide configuration properties during Ecosystem installation.
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:12]: yes I noticed some inconsistencies so I changed to Ecosystem in the revamp work so will stick with that. have changed during to after.

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+---
+path: "/docs/ecosystem/cps"
+title: "Managing automated test runs"
+---
+
+[Retrieving properties](#retrieving)<br>
+[Setting properties](#setting)<br>
+[Deleting properties](#deleting)<br>
+
+The ability to retrieve, set, and delete properties held in the configuration properties store (CPS) directly into an Ecosystem by using the Galasa CLI makes it easier for testers to set parameters and credentials on the Ecosystem for tests to read and use at runtime.
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:10]: oh yes good idea. done.

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+The example commands assume that `GALASA_BOOTSTRAP` environment variable is set, so the `--bootstrap` flag is not required in the command. 
+
+## <a name="retrieving"></a>Retrieving properties from a namespace 
+
+Use the `galasactl properties get` command to read CPS properties and values from a specified namespace in the Galasa Ecosystem to verify that the properties exist and are set correctly. You can filter the properties that are returned by using the property name (to return a single property),  or by using prefixes, suffixes, and infixes to return a subset of properties that match the provided criteria.
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:32]: removed extra space

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+### Returned properties
+
+
+Properties are returned in summary format. For example:
+
+```
+namespace name                value
+framework test.batch1.stream  value1
+framework test.batch2.stream  value2
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:83]: Will change

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+galasactl properties set --namespace docker --name docker.engine.REMOTE.hostname --value 103.67.89.6
+galasactl properties set --namespace docker --name docker.engine.REMOTE.port --value 2987
+galasactl properties set --namespace docker --name docker.engine.REMOTE.max.slots --value 20
+```
+
+You can also use `galasactl properties set` command to set a repository, OBR, and test catalog property for a test stream. For example, you can set up a test stream called `EXAMPLE` in the `framework` namespace with the following properties:   
+
+```
+framework.test.streams SIMBANK,EXAMPLE
+framework.test.stream.EXAMPLE.repo http://<your-maven-repo-hostname>/repository/maven
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:153]: oh yes that is a good idea

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+To retrieve a specific property from the `framework` namespace, specify the property name in the command by using the `-–name` flag. For example:
+
+```galasactl properties get --namespace framework --name propertyName```
+ 
+
+*Note:* The `-–name` flag cannot be used in conjunction with the `–suffix`, `--prefix`, or `-–infix` flags.
+
+### Retrieve a subset of properties in a namespace
+
+To filter the properties that are returned, without specifying the property name, use the `–prefix`, `–suffix`, and ``--infix`` flags. You can specify more than one `--infix` value by using a comma separated list. For example, to return properties in the `framework` namespace, that start with `test`, end with `stream`, and contain `batch` and `production` use the following command:
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:52]: yes that is a good idea. will change it.

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:83]: have also updated the example showing the sort order of returned results

**Review by eamansour — APPROVED** [review]

**Pull request merged**